### ISSUE TITLE
Please bundle a gruntfile

### ISSUE THREAD

**Issue by extemporalgenome** [Xe/LWALL-STUDEN-BALA#1]

**Comment by Xe** [comment]
Yes, that is definitely a solid goal for our MVP first round of fundraising from Z Combinator. I will assign someone to work on this critical task for the business logic.
